src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import duke.task.TaskList;
+import duke.util.Parser;
+import duke.util.SaveTasks;
+import duke.util.Ui;
+
+import java.nio.file.Paths;
+
+public class Duke {
+    private static final String DIR = "data";
+    private static final String FILENAME = "duke.txt";
+    private static final String FILEPATH = String.valueOf(Paths.get(DIR, FILENAME));
+    private static final String LINE = "\n----------------------------------------------------------------\n";
+    private Ui ui;
+    private SaveTasks savedTasks;
+    private TaskList taskList;
+
+    /**
+     * Constructor for Duke.
+     *
+     * @param fileDir the directory of an existing text database
+     * @param filePath the path to an existing text database
+     */
```

Comment:
The javadocs comment should describe what the class represents, not just the constructor